CHANGELOG.md

```diff
@@ -409,3 +409,6 @@
 - Fix a bug that caused record accessors for private types to not be completed
   by the LSP, even when in the same module.
   ([Ameen Radwan](https://github.com/Acepie))
+
```

Comment:
Looks like you added this to the wrong place. It's currently under the v1.4.1 changelog, it should be under `Unreleased` at the top.